### ISSUE TITLE
Podcast streams randomly/unpredictable end prematurely

### ISSUE THREAD

**Issue by duesenklipper** [LMS-Community/slimserver#494]
Hi,

For quite some time I've been having issues with podcast streaming. Longer podcasts will randomly just end long before the show is actually done. I haven't managed to pinpoint it to a particular time in the stream, nor to a particular podcast.

Most recently (as in just now) it happened with the WRINT podcast (http://www.wrint.de/feed/podcast/), episode WR1170.

It sometimes feels like it is happening with "regular" streams (like from di.fm) as well, but those recover quickly, possibly due to there being several streams in the playlists.

The problem with the podcasts makes it nearly impossible to use the Squeezeboxes in my house for listening to podcasts though.

I didn't see anything that looked relevant to me in any of the logs, and I'm not even quite sure how to debug this. Any ideas?

Thanks
Carl-Eric

**Comment by philippe44** [comment]
Try to use my "reliable" plugin, it's made precisely for that

**Comment by duesenklipper** [comment]
That seems to work very well, thank you! This should really be incorporated into LMS itself.

**Comment by philippe44** [comment]
We have added a feature in LMS 8.2 nightly that should solve the problem as well, without needing "reliable". You'll see an option for caching HTTP/HTTPS in the Settings/Advanced/Performances

**Comment by aq18507** [comment]
Thanks for including this in version 8.2 and thanks so much for your work on the LMS. I am still having the same issue with the connection dropping - that sead, it only does it certain podcasts predominantly from Swiss radio. I tried to find the cashing option you mentioned in the previous comment. I cannot find it neither in version 8.2 and 8.3 to which I just upgraded. I tried to install the "Reliable" plugin but it says that it is not compatible with versions 8.2 and 8.3.
